tox.ini

```diff
@@ -192,14 +192,16 @@ commands =
 [testenv:docs]
 deps =
   -c dev-requirements.txt
+  -c docs-requirements.txt
   sphinx
   sphinx-rtd-theme
   sphinx-autodoc-typehints
-  opentracing~=2.2.0
-  Deprecated>=1.2.6
-  thrift>=0.10.0
-  pymongo ~= 3.1
-  flask~=1.0
+  # Required by ext packages
+  opentracing
+  Deprecated
+  thrift
+  pymongo
+  flask
```

Comment:
are these still needed here if they're in the docs-requirements file?

Reply:
docs-requirements is being used as a [constraints file](https://pip.pypa.io/en/stable/user_guide/#constraints-files), so we still need the packages (just not the versions) here.